src/molecules/Dialog/Dialog.stories.tsx

```diff
@@ -223,3 +223,52 @@ export const ColorVariations: Story = {
     ],
   },
 };
+
+export const LongDialogWithContentMaxHeight: Story = {
+  args: {
+    title: 'Left and right dialog!',
```

Comment:
Ultra nitpick c:
Should this title be Long dialog with ContentMaxHeight? :D

Reply:
Pushed a commit